### PR TITLE
Add flight speed

### PR DESCRIPTION
**Pull request by Sergittos**

### CODE REVIEW

**Review by TwistedAsylumMC — CHANGES_REQUESTED** [review]
Thanks for the PR! Aside from the one comment, would you also be willing to extend this to also send the player's speed in the UpdateAbilities packet, assuming that doesn't cause any extra issues just so we can eliminate the TODO completely. If it's not as simple then I'm happy to merge this after the type is changed.

File: server/session/controllable.go [hunk 1]
```diff
@@ -32,7 +32,9 @@ type Controllable interface {
 	SetHeldItems(right, left item.Stack)
 
 	Move(deltaPos mgl64.Vec3, deltaYaw, deltaPitch float64)
+
 	Speed() float64
+	FlightSpeed() float32
```
[the remark this answers is not in the document]
Comment (Sergittos) [reply server/session/controllable.go:37]: Sure

**Comment by Sergittos** [comment]
> Thanks for the PR! Aside from the one comment, would you also be willing to extend this to also send the player's speed in the UpdateAbilities packet, assuming that doesn't cause any extra issues just so we can eliminate the TODO completely. If it's not as simple then I'm happy to merge this after the type is changed.

Added it, after some tests it seems that the walk speed only modifies the player's fov.

**Comment by DaPigGuy** [comment]
Sorry, git mishap happened

Merged manually as b4e2e0d7568add07c1d0e9c2f7bd59f24245cea7

**Pull request closed without merging**